test/gator/expand/test.bats

```diff
@@ -52,8 +52,11 @@ test_dir () {
  test_dir "expand-with-ns" 0
 }
 
+# initially we expected this scenario to fail but now we rely on 
```

Comment:
Code comments should not explain historical evolution of code, they should make sense with the code as-is.

TL;DR, drop the "initially we..." bit, fine to talk about us only expecting errors if a matcher requires the namespace be present.